src/TurbulenceConvection/closures/entr_detr.jl

```diff
@@ -219,3 +219,35 @@ function compute_entr_detr!(
         end
     end
 end
+
+##### Compute entr detr
+function compute_entr_detr!(
+    state::State,
+    grid::Grid,
+    edmf::EDMFModel,
+    εδ_closure::ConstantEntrDetrModel,
+)
+    FT = float_type(state)
+    N_up = n_updrafts(edmf)
+    aux_up = center_aux_updrafts(state)
+
+    @inbounds for i in 1:N_up
+        @inbounds for k in real_center_indices(grid)
+            if aux_up[i].area[k] > 0.0
+                # Compute fractional and turbulent entrainment/detrainment
+                aux_up[i].entr_sc[k] = FT(0.001)
+                aux_up[i].detr_sc[k] = FT(0)
```

Comment:
I would suggest setting the detrainment to a non-zero value as well, perhaps the same as entrainment.